pkg/draft/manifest/manifest.go

```diff
@@ -25,14 +25,15 @@ type Manifest struct {
 
 // Environment represents the environment for a given app at build time
 type Environment struct {
-	Name         string   `toml:"name,omitempty"`
-	BuildTarPath string   `toml:"build_tar,omitempty"`
-	ChartTarPath string   `toml:"chart_tar,omitempty"`
-	Namespace    string   `toml:"namespace,omitempty"`
-	Values       []string `toml:"set,omitempty"`
-	Wait         bool     `toml:"wait"`
-	Watch        bool     `toml:"watch"`
-	WatchDelay   int      `toml:"watch_delay,omitempty"`
+	Name          string   `toml:"name,omitempty"`
+	BuildTarPath  string   `toml:"build-tar,omitempty"`
+	ChartTarPath  string   `toml:"chart-tar,omitempty"`
+	Namespace     string   `toml:"namespace,omitempty"`
+	Values        []string `toml:"set,omitempty"`
+	AutoConnect   bool     `toml:"auto-connect"`
+	Watch         bool     `toml:"watch"`
+	WatchDelay    int      `toml:"watch-delay,omitempty"`
+	OverridePorts string   `toml:"override-ports,omitempty"`
```

Comment:
This should be an array of strings instead of a string. That would make it easier to parse!